app/org/sagebionetworks/bridge/play/controllers/ParticipantController.java

```diff
@@ -115,12 +120,6 @@ public Result updateParticipant(String userId) {
             throw new BadRequestException("ID in JSON does not match email in URL.");
         }
         participantService.updateParticipant(study, session.getParticipant().getRoles(), userId, participant);
-        
-        // Push changes to the user's session, including consent statuses.
-        CriteriaContext context = new CriteriaContext.Builder()
-                .withStudyIdentifier(study.getStudyIdentifier()).build();
-        session = authenticationService.updateSession(study, context, userId);
-        updateSession(session);
```

Comment:
Yes, this can cause errors because we're updating another user's session when we don't know their app version, and this could be used to determine consent status. I would rather that administrators purge the user's session if they want the user to pick up the changes, and that will force the user to sign in again.